plugins.json

```diff
@@ -4,7 +4,7 @@
     "author":"MassaLabs",
     "logo": "logo.svg",
```

Comment:
Need to change the logo here. What's the best way to provide you a new logo?

Reply:
should we create a new issue for that?

Reply:
I don't know @Pierregvx let us know please what's the best way here.

Reply:
we need to change https://github.com/massalabs/station-module-store/blob/main/assets/node-manager/logo.svg and do a pr

Reply:
Ok so let's close this thread and address this change in another PR as suggested by Nathan.
@Pierregvx can you open the PR please? Sending you the logo now.